tests/run.js

```diff
@@ -28,13 +28,15 @@ for (const language in testSuite) {
 			for (const filePath of testFiles) {
 				const fileName = path.basename(filePath, path.extname(filePath));
 
-				it("– should pass test case '" + fileName + "'", function () {
-					if (path.extname(filePath) === '.test') {
-						TestCase.runTestCase(language, filePath, update ? 'update' : insert ? 'insert' : 'none');
-					} else {
-						TestCase.runTestsWithHooks(language, require(filePath));
-					}
-				});
+				if (fileName[0] != '.') {
```

Comment:
Please do not modify the test suite.

Why did you modify this in the first place?

Reply:
Ah, sorry, I should have explained. I was editing one of the test files with vim, and the test runner was also trying to run my vim swap files, with errors like:

```
  1) Testing language 'ruby'
       – should pass test case '.string_feature.test':
     .../tests/languages/ruby/.string_feature.test.swp:1
b0VIM 8.2
      ^^^

SyntaxError: Unexpected number
      at wrapSafe (node:internal/modules/cjs/loader:1024:16)
      at Module._compile (node:internal/modules/cjs/loader:1072:27)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
      at Module.load (node:internal/modules/cjs/loader:973:32)
      at Function.Module._load (node:internal/modules/cjs/loader:813:14)
      at Module.require (node:internal/modules/cjs/loader:997:19)
      at require (node:internal/modules/cjs/helpers:92:18)
      at Context.<anonymous> (tests/run.js:35:44)
      at processImmediate (node:internal/timers:463:21)
```

So I made it skip hidden files, i.e. files starting with a dot (at least in *nix and macOS environments).

Reply:
I see. This is definitely a problem on our side.

I'll quickly fix the issue. Please revert this change and sync your fork.

Reply:
The problem has been fixed.